### ISSUE TITLE
.destroy function

### ISSUE THREAD

**Issue by jeromeetienne** [jeromeetienne/gowiththeflow.js#3]
- important to destroy the timerout

**Comment by jeromeetienne** [comment]
see .destroy() in this cut/paste version 

``` javascript
THREEx.Preloader.Flow   = function(){
    var self, stack = [], timerId = setTimeout(function(){ timerId = null; self._next(); }, 0);
    return self = {
        destroy : function(){ timerId && clearTimeout(timerId); },
        par : function(callback, isSeq){
            if(isSeq || !(stack[stack.length-1] instanceof Array)) stack.push([]);
            stack[stack.length-1].push(callback);
            return self;
        },seq   : function(callback){ return self.par(callback, true);  },
        _next   : function(err, result){
            var errors = [], results = [], callbacks = stack.shift() || [], nbReturn = callbacks.length, isSeq = nbReturn == 1;
            callbacks && callbacks.forEach(function(fct, index){
                fct(function(error, result){
                    errors[index]   = error;
                    results[index]  = result;       
                    if(--nbReturn == 0) self._next(isSeq?errors[0]:errors, isSeq?results[0]:results)
                }, err, result)
            })
        }
    }
};
```

**Comment by jeromeetienne** [comment]
fixed in 794ca731d1e1442c99aaf674f2610cc5c185a2ef
